package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "zenml-studio",
+  "publisher": "zenml",
+  "displayName": "ZenML Studio",
```

Comment:
@strickvl what do you think about the name?

Reply:
Probably too close to DVC who have a 'DVC Studio' product. I think we can just call it "ZenML". Simple.

Reply:
oh :) I remember that there is a pre-existing extension called zenml (https://marketplace.visualstudio.com/items?itemName=Ziphil.zenml)... but not sure if this would clash with ours at all..

Reply:
It shouldn't clash as the "publisher" field is different. The extension identifier for ours would be "zenml.zenml" or "ZenML.zenml" if we want to, while that one is "Ziphil.zenml".